.gitignore

```diff
@@ -3,6 +3,7 @@
 /.java-version
 /.karaf
 /.yardoc
+/.vscode/
```

Comment:
plz to not add this. either use ~/.gitignore, or .git/info/excludes. Unless an editor or a tool is the preferred tool of a repository, you shouldn't add its files to .gitignore. It's a never-ending battle because every developer has a slightly different workflow.

Reply:
Thanks! TIL .git/info/excludes